ghost/core/core/server/services/posts/stats/post-stats.js

```diff
@@ -0,0 +1,31 @@
+class PostStats {
+    /**
+     * @param {Object} config
+     * @param {Object} config.db - an instance holding knex connection to the database
+     * @param {Boolean} config.isSQLite - flag identifying if storage is connected to SQLite
+     */
+    constructor({db, isSQLite}) {
+        this._db = db;
```

Comment:
You might consider passing a knex instance here instead - also no biggie

Reply:
Also don't forget you can now use private members, so no need for underscore prefixes! `this.#db = db;` etc...

Reply:
Ah, ok. I honestly copies the members stats code here which was using exactly that approach.

Reply:
S'all good - don't need to change it if you don't want!